<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorView</class>
 <widget class="QWidget" name="SensorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelMesureValue">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMesure">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveMesureValue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStopMesure">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
